app/components/pages-menu/component.js

```diff
@@ -4,6 +4,20 @@ export default Ember.Component.extend({
     style: Ember.computed('layer.settings.values.color', function(){
         return Ember.String.htmlSafe('color: ' + (this.get('layer.settings.values.color') || '#333') + '; ');
     }),
+    sticky: Ember.observer('layer.settings.values.StickToTop' , function() {
+        let topOfNav = $('.pages-menu').offset().top;
```

Comment:
In general selectors for jquery ('.pages-menu') need to be precise otherwise it will cause issues down the line, I assume there won't be another class name like this on the page right?